rmf_task_msgs/srv/GetTask.srv

```diff
@@ -0,0 +1,15 @@
+# Query list of submitted tasks | Get service call
```

Comment:
Maybe `GetTasks` or `GetTaskList` would be a more appropriate service name?

Reply:
Renamed to `GetTaskList` https://github.com/osrf/rmf_core/pull/217/commits/0f57a1049b07dbd99c304bfc0cec0eb87d5e2588

@tanyouliang95 heads up to update the Dispatcher node with this new service name